plugins/SSH.koplugin/main.lua

```diff
@@ -141,8 +144,8 @@ function SSH:addToMainMenu(menu_items)
                 text = _("SSH public key"),
                 callback = function()
                     local info = InfoMessage:new{
-                        timeout = 5,
-                        text = _("Put your public SSH keys in settings/SSH/authorized_keys"),
+                        timeout = 60,
+                        text = _("Put your public SSH keys in "..lfs.currentdir().."/settings/SSH/authorized_keys"),
```

Comment:
You should use the `DataStorage:getFullDataDir()` abstraction instead of lfs.currentdir() directly. On Kindle and Kobo it doesn't matter because the end result is the same, probably also on PocketBook, but as a general code quality principle I think it's a bad idea to let assumptions that are known to break down on other platforms sneak in.

Reply:
Sure, it easy to do.